index.js

```diff
@@ -23,7 +25,18 @@ fastify.register(
 )
 
 // CORS
-fastify.register(require('fastify-cors'))
+fastify.register(require('fastify-cors'), {
+  methods: ['GET'],

```

Comment:
If we have `fastifyCorsOptions` in the config file, our change here would be:
```JavaScript
fastify.register(require('fastify-cors'), fastifyCorsOptions)
```

Reply:
cool yeah makes perfect sense, let me make the changes